lidnd/utils/campaigns.ts

```
@@ -1,12 +1,21 @@
+    return new Date(a.created_at).getTime() - new Date(b.created_at).getTime();
+  }
+  return 0;
+}
+
+export function encounterFromUrl(
+  campaign: { encounters: Encounter[] },
+  url: string
+) {
+  const encounterIndex = url.split("/").at(4);
+  const indexToFind = parseInt(encounterIndex ?? "");
+  if (isNaN(indexToFind)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)